### ISSUE TITLE
Problem with Jetpack 3.1 installation

### ISSUE THREAD

**Issue by moiseevigor** [jetsonhacks/installTensorFlowTX2#9]
I've had one problem with Jetpack 3.1, solved it with this small change

```
diff --git a/setTensorFlowEV.sh b/setTensorFlowEV.sh
index 9b21dfa..0e6c82c 100755
--- a/setTensorFlowEV.sh
+++ b/setTensorFlowEV.sh
@@ -27,7 +27,7 @@ TF_CUDA_VERSION=8.0
 default_cuda_path=/usr/local/cuda
 CUDA_TOOLKIT_PATH=$default_cuda_path
 # cuDNN
-TF_CUDNN_VERSION=5.1.10
+TF_CUDNN_VERSION=6.0.21
 default_cudnn_path=/usr/lib/aarch64-linux-gnu
 CUDNN_INSTALL_PATH=$default_cudnn_path
 # CUDA compute capability
```

the latest Jetpack solves the annoying gpu memory allocation problem https://devtalk.nvidia.com/default/topic/1013464/jetson-tx2/gpu-out-of-memory-when-the-total-ram-usage-is-2-8g/post/5191096/#5191096

**Comment by Goddard** [comment]
Here is a repo you can use until it is fixed for anyone that finds this...

https://github.com/Goddard/installTensorFlowTX2

**Comment by jetsonhacks** [comment]
Should be fixed with commit: 10b022462ecbf10c534964cdee5530c3a182907a
Thank you for your fix.
